packages/shorebird_cli/lib/src/archive_analysis/android_archive_differ.dart

```diff
@@ -22,33 +14,19 @@ abstract class AndroidArchiveDiffer extends ArchiveDiffer {
     }
 
     return assetsDiff.changedPaths
```

Comment:
I guess this is also assetsDiff.changePaths.map(p.basename).toSet().difference(ArchiveDiffer.assetFileNamesToIgnore.toSet()).isEmpty(), but that's no shorter. 🤣